<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion personnel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1091</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  QPushButton {
         border: 2px solid #8f8f91;
         border-radius: 6px;
         background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                           stop: 0 #f6f7fa, stop: 1 #dadbde);
         min-width: 80px;
     }

     QPushButton:pressed {
         background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                           stop: 0 #dadbde, stop: 1 #f6f7fa);
     }

     QPushButton:flat {
         border: none; /* no border for a flat push button */
     }

     QPushButton:default {
         border-color: navy; /* make the default button prominent */
     }
QPushButton::hover
{
	background-color: #09c1eb;

}
QPushButton::pressed
{
	background-color: #3d4ef2;

}
 QLineEdit,QDateTimeEdit,QPlainTextEdit {
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: #e0ebeb	;
     selection-background-color: darkgray;
 }
QComboBox {
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: #e0ebeb	;
     selection-background-color: darkgray;
 }
   QTabWidget::pane { /* The tab widget frame */
         border-top: 2px solid #C2C7CB;
     }

     QTabWidget::tab-bar {
         left: 5px; /* move to the right by 5px */
     }

     /* Style the tab using the tab sub-control. Note that
         it reads QTabBar _not_ QTabWidget */
     QTabBar::tab {
         background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                     stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                     stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
         border: 2px solid #C4C4C3;
         border-bottom-color: #C2C7CB; /* same as the pane color */
         border-top-left-radius: 4px;
         border-top-right-radius: 4px;
         min-width: 8ex;
         padding: 2px;
         width: 140px;
     }

     QTabBar::tab:selected, QTabBar::tab:hover {
         background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                     stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                     stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
     }

     QTabBar::tab:selected {
         border-color: #9B9B9B;
         border-bottom-color: #C2C7CB; /* same as pane color */
     }

     QTabBar::tab:!selected {
         margin-top: 2px; /* make non-selected tabs look smaller */
     }
QTabBar::tab:selected
{
	background-color: /*qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(54, 197, 177, 255),stop:1 rgba(11, 135, 111, 255)); */#3a90a6;
	color: #fff;

}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Les employ√©s</string>
     </attribute>
     <widget class="QTableView" name="tab_personnel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>791</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*QTableView Top left style */
QTableView QTableCornerButton::section {
   /*  background: red;
    border: 2px outset red;*/
    color: red;
    background-color: rgb(64, 64, 64);
    border: 5px solid #f6f7fa;
    border-radius:0px;
    border-color: rgb(64, 64, 64);
 }

 QTableView {
         Color: white; /* text color in the table */
         Gridline-color: black; /*The inner frame color*/
         Background-color: rgb(108, 108, 108); /* background color in the table*/
    alternate-background-color: rgb(64, 64, 64);
         Selection-color: white; /* The text color of the selected area*/
         Selection-background-color: rgb(77, 77, 77); /*The background color of the selected area*/
    border: 2px groove gray;
    border-radius: 0px;
    padding: 2px 4px;
}

QHeaderView {
    color: white;
    font: bold 10pt;
    background-color: rgb(108, 108, 108);
    border: 0px solid rgb(144, 144, 144);
    border:0px solid rgb(191,191,191);
    border-left-color: rgba(255, 255, 255, 0);
    border-top-color: rgba(255, 255, 255, 0);
    border-radius:0px;
    min-height:29px;
}

QHeaderView::section {
    color: white;
    background-color: rgb(64, 64, 64);
    border: 5px solid #f6f7fa;
    border-radius:0px;
    border-color: rgb(64, 64, 64);
} </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>441</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>460</y>
        <width>341</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Supprimer</string>
      </property>
      <widget class="QSplitter" name="splitter_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>321</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>CIN :</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_supp"/>
       </widget>
       <widget class="QPushButton" name="supprimer_pb">
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>70</y>
        <width>271</width>
        <height>381</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter</string>
      </property>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>300</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter_18">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>CIN :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_id"/>
      </widget>
      <widget class="QSplitter" name="splitter_19">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Nom :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_nom"/>
      </widget>
      <widget class="QSplitter" name="splitter_20">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Mail :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_prenom"/>
      </widget>
      <widget class="QSplitter" name="splitter_21">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Tel :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_tel"/>
      </widget>
      <widget class="QSplitter" name="splitter_22">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Service :</string>
        </property>
       </widget>
       <widget class="QComboBox" name="le_service">
        <item>
         <property name="text">
          <string>Cuisine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Serveur</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Caissier</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_23">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Mot de passe :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_mdp">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Absence</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>401</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Ajouter </string>
      </property>
      <widget class="QPushButton" name="Ajouter_absence">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>190</y>
         <width>84</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>291</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>ID :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="id_absence"/>
      </widget>
      <widget class="QSplitter" name="splitter_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>291</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Nom :</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_nom"/>
      </widget>
      <widget class="QSplitter" name="splitter_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>291</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Motif :</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_motif">
        <item>
         <property name="text">
          <string>Vacances</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Malade</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Autre</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>291</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Date : </string>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="date"/>
      </widget>
     </widget>
     <widget class="QTableView" name="tab_absence">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>40</y>
        <width>571</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*QTableView Top left style */
QTableView QTableCornerButton::section {
   /*  background: red;
    border: 2px outset red;*/
    color: red;
    background-color: rgb(64, 64, 64);
    border: 5px solid #f6f7fa;
    border-radius:0px;
    border-color: rgb(64, 64, 64);
 }

 QTableView {
         Color: white; /* text color in the table */
         Gridline-color: black; /*The inner frame color*/
         Background-color: rgb(108, 108, 108); /* background color in the table*/
    alternate-background-color: rgb(64, 64, 64);
         Selection-color: white; /* The text color of the selected area*/
         Selection-background-color: rgb(77, 77, 77); /*The background color of the selected area*/
    border: 2px groove gray;
    border-radius: 0px;
    padding: 2px 4px;
}

QHeaderView {
    color: white;
    font: bold 10pt;
    background-color: rgb(108, 108, 108);
    border: 0px solid rgb(144, 144, 144);
    border:0px solid rgb(191,191,191);
    border-left-color: rgba(255, 255, 255, 0);
    border-top-color: rgba(255, 255, 255, 0);
    border-radius:0px;
    min-height:29px;
}

QHeaderView::section {
    color: white;
    background-color: rgb(64, 64, 64);
    border: 5px solid #f6f7fa;
    border-radius:0px;
    border-color: rgb(64, 64, 64);
} </string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>401</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Supprimer </string>
      </property>
      <widget class="QPushButton" name="Supprimer_absence">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>100</y>
         <width>84</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>281</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>ID :</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_id3"/>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>380</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>T√©l√©charger</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Mailing</string>
     </attribute>
     <widget class="QSplitter" name="splitter_2">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>21</y>
        <width>531</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Smtp-server:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="server">
       <property name="text">
        <string>smtp.gmail.com</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_3">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>52</y>
        <width>531</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Server port:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="port">
       <property name="text">
        <string>465</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_4">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>83</y>
        <width>531</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="uname"/>
     </widget>
     <widget class="QSplitter" name="splitter_5">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>114</y>
        <width>531</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="paswd">
       <property name="inputMask">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_6">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>145</y>
        <width>531</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Recipant to:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="rcpt"/>
     </widget>
     <widget class="QSplitter" name="splitter_7">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>176</y>
        <width>531</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Subject:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="subject"/>
     </widget>
     <widget class="QSplitter" name="splitter_14">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>207</y>
        <width>531</width>
        <height>192</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Message:</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="msg"/>
     </widget>
     <widget class="QSplitter" name="splitter_15">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>415</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="sendBtn">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
      <widget class="QPushButton" name="exitBtn">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1127</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
